source/ed/docker_tests/CMakeLists.txt

```diff
@@ -38,8 +47,7 @@ target_link_libraries(ed_integration_test data georef types utils ${BOOST_LIBS}
 add_custom_target(run_test DEPENDS datanav_files ed_integration_test)
 add_custom_command(TARGET run_test
                    COMMAND ed_integration_test
-                   ARGS --log_format=XML --log_sink=results_ed_integration_test.xml --log_level=all --report_level=no --ntfs_file=${DATA_NAV_TO_CREATE})
-
+                   ARGS --log_format=XML --log_sink=results_ed_integration_test.xml --log_level=all --report_level=no ${TEST_CLI_PARAMS})
```

Comment:
it's argument forwarded to the exe, I don't know how to tell cmake to consider it one line and I don't really want to dig into cmake :wink: